mne/gui/_coreg_gui.py

```diff
@@ -1194,12 +1212,12 @@ class CoregFrame(HasTraits):
     guess_mri_subject = DelegatesTo('model')
 
     # Omit Points
-    distance = Float(5., label="Distance [mm]", desc="Maximal distance for "
-                     "head shape points from MRI in mm")
-    omit_points = Button(label='Omit Points', desc="Omit head shape points "
+    distance = Float(5., desc="Maximal distance for head shape points from "
```

Comment:
Since traits converts `desc` to `"Specifies " + desc` you could start with a lowercase letter to avoid a tooltip like "Specifies Maximum distance ..."